### PR TITLE
DB migrations

### PR DESCRIPTION
**Pull request by chelsea-EYDS**

### CODE REVIEW

File: apps/backend/src/database/queries/locations.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
+export const locations = `
+INSERT INTO public."location" (merchant_id,location_id,ax_merchant_id,pt_location_id,office_name) VALUES
```
Comment (chelsea-EYDS) [inline apps/backend/src/database/queries/locations.ts:2]: Included to use as seed data for testing

File: apps/backend/src/database/queries/locations.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
+export const locations = `
```
Comment (chelsea-EYDS) [inline apps/backend/src/database/queries/locations.ts:1]: remove

File: apps/backend/src/database/config.ts [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    PaymentMethodEntity,
+    PaymentEntity,
+    TransactionEntity,
+    POSDepositEntity,
+    CashDepositEntity,
+    LocationView,
+    MasterLocationDataEntity
+  ],
+  migrations: [__dirname + '/migrations/**/*{.ts,.js}'],
+  synchronize: process.env.NODE_ENV === 'test' ?? false
```
Comment (vyasworks) [inline apps/backend/src/database/config.ts:38]: Why should tests run with `synchronize`?

File: apps/backend/src/database/migrations/1674455627553-migration.ts [hunk 4]
```diff
@@ -0,0 +1,47 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
```
Comment (vyasworks) [inline apps/backend/src/database/migrations/1674455627553-migration.ts:1]: Is this the only migration? Why is there only alter DDLs?

**Review by vyasworks — CHANGES_REQUESTED** [review]

File: apps/backend/src/reconciliation/entities/location-view.entity.ts [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+   * Materialized Location View
+   */
+}
+
+@ViewEntity({
+  materialized: true,
+  name: 'location_view',
+  database: 'bcpcc',
+  schema: 'public',
+  expression: `SELECT id, "GARMS Location", "Merchant ID", "Location", "description", "Type"  FROM master_location_data
```
Comment (vyasworks) [inline apps/backend/src/reconciliation/entities/location-view.entity.ts:15]: this doesn't seem right

File: apps/backend/src/reconciliation/entities/location-view.entity.ts [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+
+{
+  /*
+   * Materialized Location View
+   */
+}
+
+@ViewEntity({
+  materialized: true,
+  name: 'location_view',
```
Comment (vyasworks) [inline apps/backend/src/reconciliation/entities/location-view.entity.ts:12]: name does not imply what this view is

**Review by vyasworks — CHANGES_REQUESTED** [review]

**Review by vyasworks — APPROVED** [review]

**Pull request merged**